src/database/models/User.ts

```diff
@@ -73,4 +76,7 @@ export class User extends Model {
     as: "sites_created",
   })
   sitesCreated?: Site[]
+
+  @HasMany(() => ReviewRequest)
+  reviewRequests?: ReviewRequest[]
```

Comment:
Can I confirm that this is referring to the review requests that this User has **created**, and not the list of review requests that this User has been selected to be a reviewer?

Reply:
actually i think this is ambiguous (didn't try) and it's a good point raised. wdyt if i changed it to `reviewsRequested` and `reviewsCreated` for clarity?

Reply:
Yup I think that would make things clearer, if it is possible to codify such a relationship in this model.

Reply:
this might be slightly annoying to do btw... @__@

Reply:
I think it should be sufficient to get the `reviewsCreated`.